src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -14,38 +15,55 @@ public class CommandResult {
     private final String feedbackToUser;
 
     /** Help information should be shown to the user. */
-    private final boolean showHelp;
+    private final boolean isShowHelp;
 
     /** The application should exit. */
-    private final boolean exit;
+    private final boolean isExit;
+
+    /** Specifies the viewing mode that the application should be in after command execution. */
+    private final ViewMode viewMode;
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp,
+                         boolean exit, ViewMode viewMode) {
```

Comment:
Nit but since we are standardising to is___, should we change the arguments to fit it as well?